app/controllers/taxon_framework_relationships_controller.rb

```diff
@@ -0,0 +1,203 @@
+class TaxonFrameworkRelationshipsController < ApplicationController
+  before_filter :authenticate_user!, :except => [:index, :show]
+  before_filter :admin_required, :only => [:new, :create, :edit, :update, :destroy]
+  before_action :set_taxon_framework_relationship, except: [:index, :new, :create]
+  
+  layout "bootstrap"
+  
+  def index
+    filter_params = params[:filters] || params
+    @relationships = filter_params[:relationships]
+    @relationships ||= TaxonFrameworkRelationship::RELATIONSHIPS.map{ |r| filter_params[r] == "1" ? r : nil }
+    @relationships.delete_if{ |r| r.blank? }
+    @relationships = @relationships.map{ |r| r.gsub( "_"," " ) }
```

Comment:
Not sure why this is happening but it breaks filtration, e.g. you can't filter by many_to_many because you end up querying on `taxon_framework_relationships.relationship IN ('many to many')`

Reply:
fixed